### PR TITLE
Update cocos documentation

### PR DESCRIPTION
**Pull request by fbugarski**
Updated HAL and Manager documentation.

### CODE REVIEW

**Review by drasko — COMMENTED** [review]
Is this still draft?

**Comment by drasko** [comment]
@fbugarski please resolve the conflicts so that we can mere this one if it is finished.

**Comment by fbugarski** [comment]
> @fbugarski please resolve the conflicts so that we can mere this one if it is finished.

@danko-miladinovic have more updates i think, and he will continue to work on this PR, it is still DRAFT.

File: manager/README.md [hunk 1]
```diff
@@ -43,9 +43,14 @@ sudo apt install qemu-kvm
 
 Create `img` directory in `cmd/manager`. Create `tmp` directory in `cmd/manager`.
 
-#### Add Vsock
+#### Add V-sock
```
Comment (drasko) [inline manager/README.md:46]: It's actually called Vsock, not V-sock

File: manager/README.md [hunk 2]
```diff
@@ -198,7 +203,7 @@ MANAGER_QEMU_KERNEL_HASH=true \
 
 ### Verifying VM launch
 
-NB: To verify that the manager successfully launched the VM, you need to open two terminals on the same machine. In one terminal, you need to launch `go run main.go` (with the environment variables of choice) and in the other, you can run the verification commands.
+NB: To verify that the manager successfully launched the VM, you need to open three terminals on the same machine. In one terminal, you need to launch the Manager test server by executing go run ./test/manager-server/main.go (with the environment variables of choice), and in the second the manager by executing go run ./cmd/manager/main.go (with the environment variables of choice). Ensure that the Manager can connect to the Manager test server by setting the MANAGER_GRPC_PORT with the port value of the Manager test server. The Manager test server is listening on the default value of the MANAGER_GRPC_PORT. In the last one, you can run the verification commands.
```
Comment (drasko) [inline manager/README.md:206]: Put commands (`go run ...`) in verbatim, and maybe even on a separate line, for better readibility.

**Review by drasko — CHANGES_REQUESTED** [review]

File: manager/README.md (hunk 1 above)
Comment (danko-miladinovic) [reply manager/README.md]: Changed.

File: manager/README.md (hunk 2 above)
Comment (danko-miladinovic) [reply manager/README.md]: Changed.

File: manager/README.md [hunk 3]
```diff
@@ -198,7 +203,15 @@ MANAGER_QEMU_KERNEL_HASH=true \
 
 ### Verifying VM launch
 
-NB: To verify that the manager successfully launched the VM, you need to open two terminals on the same machine. In one terminal, you need to launch `go run main.go` (with the environment variables of choice) and in the other, you can run the verification commands.
+NB: To verify that the manager successfully launched the VM, you need to open three terminals on the same machine. In one terminal, you need to launch the Manager test server by executing (with the environment variables of choice):
+
+`go run ./test/manager-server/main.go`
```
[the remark this answers is not in the document]
Comment (danko-miladinovic) [reply manager/README.md]: Changed

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**